### ISSUE TITLE
Include examples in repository?

### ISSUE THREAD

**Issue by octalmage** [octalmage/robotjs#46]
I'm wondering if we should include some examples in the repository. I always like it when projects do this, it makes it easy to test the library and see what it can do. But it also clutters the repository, which I don't want to do. 

Thoughts?

**Comment by Prof-Dan** [comment]
You could always make a new repository for the examples...

**Comment by Deltatiger** [comment]
I agree with @Prof-Dan. We should move all examples and demos to a new repository. That would make maintaining very easy.
Also creating a unit test for this would also be good. Just a thought. Something like regression would be great for testing if any new bugs popped up.

**Comment by Prof-Dan** [comment]
Yes, making unit tests for every individual feature would make tracking down bugs a lot easier, and also making an example for each individual feature (moveMouse, typeString, etc.) would make testing for bugs easier as well.

**Comment by octalmage** [comment]
I've got tests in the "test" directory, just working on a way to test things like typing and clicking. I'm making a testing tool that will open a GUI with buttons and inputs for those features.

**Comment by octalmage** [comment]
But I agree there should be a separate repository for the examples. I'll set this up when I get back to Austin. I for sure want to make a screen color picker. And something fun like this:

https://twitter.com/octalmage/status/561312155341824000

**Comment by octalmage** [comment]
Closing stale issues as part of a cleanup. If this is still an issue, please reopen and I'll take a look. Thanks!
